person.rb

```diff
@@ -3,14 +3,15 @@
 class Person < Nameable
   # setters and getters
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
```

Comment:
- [ ] `Getters and Setters` for `@rentals` should be present which means `attr_accessor` and not `attr_reader`, please kindly fix that 🙏🏾 
- [ ] Also you should have a method for adding rentals should be present, and the person itself should be sent as a parameter to create the new rental achieving the `has-many` association, kindly add the method 🙃 😁